.github/workflows/continuous-integration.yaml

```diff
@@ -36,8 +39,8 @@ jobs:
       uses: actions/cache@v2
       with:
         path: /tmp/m2
-        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-        restore-keys: ${{ runner.os }}-m2
+        key: ${{ runner.os }}-m2-${{ steps.cachekeystep.outputs.pomcachekey }}
+        restore-keys: ${{ runner.os }}-m2-
```

Comment:
I don't understand the significance of the extra `-` at the end of this line. Could you elaborate?